common-core-open/src/main/java/edu/isi/nlp/files/KeyValueSources.java

```diff
@@ -92,4 +93,26 @@ private KeyValueSources() {
     }
     return new ZipKeyValueSource(zipFile, ret.build());
   }
+
+  @Nonnull
+  public static ImmutableKeyValueSource<Symbol, ByteSource> fromVistaUtilsZipKeyValueSource(
+      final ZipFile zipFile) {
+    // read "__keys" file
+    ZipEntry keysEntry = zipFile.getEntry(("__keys"));
```

Comment:
Check and give an informative error message if the `__keys` entry is not present.